packages/yew/src/platform/rt_wasm_bindgen/sync/mpsc/mod.rs

```diff
@@ -0,0 +1,62 @@
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+use futures::channel::mpsc as imp;
```

Comment:
We can use tokio's channels with wasm-bindgen

> The mpsc channel does not care about which runtime you use it in, and can be used to send messages from one runtime to another. It can also be used in non-Tokio runtimes.

From [`tokio::sync::mpsc` docs](https://docs.rs/tokio/latest/tokio/sync/mpsc/index.html)

Why not use those? The sync module can be simplified (no rt specific implementation needed) and the end result will be faster channels. Or is shipping tokio's channels to browser too expensive?

Reply:
Tokio does not compile on wasm yet.

I have restored the specialisation on mpsc channels.
If the ultimate goal is to move towards Tokio's implementation once it becomes available in wasm, it might be better to make `sync` private and move to tokio directly.

Reply:
Tokio does compile for wasm.

I checked out commit d656fc71

```toml
[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
tokio = { version = "1.19", features = ["full"] }
```
This is the problematic code. For some reason, this is affecting `cargo build --target wasm32-unknown-unknown`. Do you have any idea why?

Reply:
The examples with ssr were still failing as they have warp and request which selects the `net` feature.

This actually have something to do with cargo resolver being defaulted to 1 when a workspace is used ignoring edition preference set at package level.